### ISSUE TITLE
Using an attribute in place of the row name?

### ISSUE THREAD

**Issue by daytonturner** [benct/lovelace-multiple-entity-row#255]
According to the docs on github, `attribute` is valid at the root level, and it seems to be accepted, but only really seems linked to the `show_state` boolean - whereby, if show_state is on, it will display an attribute instead of the state.

I'm wondering, is there a way to display an attribute in place of `name` ?

**Comment by ildar170975** [comment]
> I'm wondering, is there a way to display an attribute in place of `name` ?

What do you mean:
- show a value of the attribute in place of the name
- show a name of the attribute in place of the name
?

**Comment by daytonturner** [comment]
Yeah, it was the 1st - show the value of an attribute in place of the name.  I am trying to display the next 4 ferry sailings and right now i am using "1st Ferry", "2nd Ferry", etc.. and realize just displaying the sailing time would be cleaner.

Bummer that it's not currently possible! Could it be implemented, or is there a reason not to?

**Comment by ildar170975** [comment]
It could be implemented by placing the row inside a `custom:config-template-card`.
I suggest to ask this question (as well as similar questions) in a Community forum in a corresponding thread and leave Github for bugs & FRs only.

**Comment by xthursdayx** [comment]
@daytonturner did you ever figure out how to get this to work? Either with multiple-entity-row or config-template card. I'm trying to figure out something similar. Thanks!

**Comment by jpettitt** [comment]
`name` is templatable as of 4.8.0, so an attribute can drive it directly: `name: "{{ state_attr(entity, 'some_attribute') }}"`. Works on the row and on additional entities. Closing.
